src/content/docs/accounts/original-accounts-billing/product-pricing/product-based-pricing.mdx

```diff
@@ -211,7 +209,7 @@ Here is a summary of the available options from your [account dropdown](/docs/ac
     * [Billing credit card email](https://docs.newrelic.com/docs/accounts-partnerships/accounts/account-billing-usage/change-credit-card-or-payment-method)
     * [License key](/docs/accounts-partnerships/accounts/account-setup/license-key)
     * [Child accounts](/docs/accounts-partnerships/accounts/account-setup/create-sub-accounts) if applicable
-    * Subscriptions to [New Relic products](https://newrelic.com/products)
+    * Subscriptions to New Relic products
```

Comment:
not sure that "products" is the right word anymore. maybe change to "capabilities"?

Reply:
I tried to just leave as much of the old pricing terminology as possible in these docs, but agree that outside the "original accounts billing" category this should be "capabilities" or "features" or similar.